CONTRIBUTING.md

```diff
@@ -226,6 +226,9 @@ You can find a large number of real-world examples of changelog files
 At release time, the entries will be merged with
 [this tool](https://github.com/phadej/changelog-d).
 
+In addition, if you're changing the .cabal file format specification you should
```

Comment:
How about cabal.project file format? That's not a part of Cabal the library?

Reply:
no, that's only a cabal-install thing (though it's based on Cabal's field grammar)